components/style/properties.mako.rs

```diff
@@ -6498,23 +6457,17 @@ fn cascade_with_cached_declarations(
                                             DeclaredValue::WithVariables { .. } => unreachable!()
                                         }, &mut error_reporter
                                     );
-                                    Arc::make_mut(&mut style_${style_struct.ident})
+                                    Arc::make_mut(&mut context.style.${style_struct.ident})
                                         .${property.ident} = computed_value;
                                 % endif
 
                                 % if property.name in DERIVED_LONGHANDS:
-                                    % if not style_struct.inherited:
```

Comment:
It’s not needed since `derive_from_*` functions don’t take a `computed_value` parameter anymore. Computed values (as known "so far") are already available through e.g. `context.style.box_.display`, whereas `context.display` was previously the *specified* value, so when the computed value was needed it had to be passed separately.